uefi/src/proto/device_path/mod.rs

```diff
@@ -975,4 +989,33 @@ mod tests {
         let owned_dp_ref = &*owned_dp;
         assert_eq!(owned_dp_ref, dp)
     }
+
+    #[test]
+    fn test_device_path_from_bytes() {
+        let mut raw_data = Vec::new();
+        let node = [0xa0, 0xb0];
+        let node_data = &[10, 11];
+        let mut dp;
```

Comment:
nit: remove the `dp` variable and directly `assert` the result of `try_from` instead